intake/catalog/tests/test_local.py

```diff
@@ -121,19 +121,19 @@ def test_use_source_plugin_from_config(catalog1):
 
 
 def test_get_dir():
-    assert get_dir("file:///path/catalog.yml") == "file:///path"
-    assert get_dir("https://example.com/catalog.yml") == "https://example.com"
+    assert get_dir("file:///path/catalog.yml") == "file:/path/"
```

Comment:
The loss of "//" is problematic here

Reply:
I reverted the previous version with keeping `://`